app/views/contexts/list_context.py

```diff
@@ -42,18 +43,18 @@ def __call__(
 
     def _build_list_items_context(
         self,
-        for_list,
-        return_to,
-        summary_definition,
-        edit_block_id,
-        remove_block_id,
-        primary_person_edit_block_id,
-        for_list_item_ids,
-    ):
+        for_list: str,
+        return_to: Optional[str],
+        summary_definition: Mapping[str, Any],
+        edit_block_id: Optional[str],
+        remove_block_id: Optional[str],
+        primary_person_edit_block_id: Optional[str],
+        for_list_item_ids: Optional[str],
```

Comment:
`for_list_item_ids` should be Optional Iterable/Sequence of strings.

Reply:
👍